docker/dev.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Change working directory to the root of the repository
+cd "${0%/*}/.."
+
+COMPOSE_FILE=docker/dev.yml
+
+# Check if 2nd argument is security
+if [ "$2" = "security" ]; then
+    export SECURITY_PLUGIN_REPO_PATH="../../wazuh-security-dashboards-plugin"
+    COMPOSE_FILE=docker/dev_security.yml
+
+fi
+
+# Common variables
+export REPO_PATH=$(pwd)
+export NODE_VERSION=$(cat .nvmrc)
+export OPENSEARCH_VERSION=$(bash dev-tools/get-version.sh)
```

Comment:
This script is removed in OpenSearch 2.6.0. It's replaced in the `package.json` by Node command instead.

Reply:
As we want the environment to be node-agnostic, using the Node command would not be a solution for use.

Instead, I'll port the bash command in `dev-tools/get-version.sh`. The method to read the OpenSearch version in the `package.json` file will diverge from OpenSearch's, but we are fine with it.

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard/pull/30/commits/ef942756ac5eacecc2fabc25d71bbd6bfa7551a6